agent/agent.go

```diff
@@ -1200,7 +1201,7 @@ func panicRecover(input *models.RunningInput) {
 		runtime.Stack(trace, true)
 		log.Printf("E! FATAL: [%s] panicked: %s, Stack:\n%s",
 			input.LogName(), err, trace)
-		log.Println("E! PLEASE REPORT THIS PANIC ON GITHUB with " +
+		log.Fatal("E! PLEASE REPORT THIS PANIC ON GITHUB with " +
```

Comment:
```suggestion
		log.Fatalln("E! PLEASE REPORT THIS PANIC ON GITHUB with " +
```

Please also remove the `defer panicRecover(input)` call in line 559 as this will never be called AFAICS....